scripts/lib/build.fish

```diff
@@ -138,7 +138,7 @@ function shutdownCcache
   if test "$USE_CCACHE" = "On"
     ccache --show-stats
   else if test "$USE_CCACHE" = "sccache"
-    sccache --stop-server; or echo "warning: cannot stop sccache"
+    sccache --stop-server 2> $INNERWORKDIR/sccache.err.log; or echo "warning: cannot stop sccache. See $INNERWORKDIR/sccache.err.log"
```

Comment:
How will it stop a not running server? we only care if we can't start it, not if it won't stop.